src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -415,6 +438,13 @@ def get_command_table(module_name=None):
         logger.debug("Loaded all modules in %.3f seconds. "
                      "(note: there's always an overhead with the first module loaded)",
                      cumulative_elapsed_time)
+    try:
+        # We always load extensions even if the appropriate module has been loaded
+        # as an extension could override the commands already loaded.
```

Comment:
Why is this?  I would expect the official packages to be reserved and for extensions to be subservient.  This seems like it could be a security issue. @troydai what do you think?